src/AaveV2FreezeReserves_20230619/V2-to-V3-MIGRATION-NEXT-STEPS.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Chaos Labs V2 to V3 Migration Next Steps
+author: Chaos Labs (@Chaos Labs)
+shortDescription: Chaos Labs V2 to V3 Migration Next Steps
```

Comment:
```suggestion
```
There's no ui using this field and most proposals just duplicate the title. Better remove it.